routing_common/transit_types.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "routing/segment.hpp"
```

Comment:
So, routing depends on routing_common or routing_common depends on routing?

Reply:
Ops... We have a problem. Fortunately it's quite easy to fix.

Reply:
Fixed.